app/views/category/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class='navbar navbar-login'>
+  <%= stylesheet_link_tag "categories" %>
+  <%= link_to 'Back', splash_screen_path, class: 'back-link' %>
+  <h2 style="color: white; background-color: transparent;">CATEGORIES</h2>
+  <%= button_to "Sign out", destroy_user_session_path, method: :delete, class: "sign_out text_white" %>
+</div>
+
+<div class="categories">
+  <% if @categories.length == 0 %>
+    <p class="empty_data">No category yet. Add a new one!!!</p>
+  <% else %>
+    <% @categories.each do |category| %>
+      <%= link_to category_expense_index_path category do %>
+        <%= render category %>
+      <% end %>
+    <% end %>
+  <% end %>
+</div>
+
+<div class="new_category_btn text_white">
+  <%= link_to "Add Category", new_category_path, class: "fcc-btn" %>
```

Comment:
- [ ] Please make sure this button appears at the bottom of the page and looks like that provided in the original design. It should have a white border surrounding the button text. According to the design guidelines, you should follow:
 - Colors
 - Typography: font face, size and weight.
 - Layout: composition and space between elements.

### Your implementation
![image](https://github.com/yosaddis/ruby-on-rails-capstone/assets/45460192/f5c6dce7-c559-4ac0-b26b-5a9faed568e8)
### Required
![image](https://github.com/yosaddis/ruby-on-rails-capstone/assets/45460192/c30c6739-b911-4031-b1d8-3f8bc2f1752b)